tests/snapshots/assets__check_file@cetz-manual.typ-0.snap

```diff
@@ -29,14 +29,12 @@ input_file: tests/assets/cetz-manual.typ
 #set par(justify: true)
 #set heading(numbering: (
   ..num,
-) => (
-  if num.pos().len() < 4 {
-    numbering(
-      "1.1",
-      ..num,
-    )
-  }
-))
+) => if num.pos().len() < 4 {
+  numbering(
+    "1.1",
+    ..num,
+  )
+})
 #show link: set text(blue)
 
```

Comment:
ideally, we should allow printer too choose between these 2 layout depending on the column width. It should first try current one, if it cannot fit, then switch to the previous one. But I think it is not possible in current pretty printer

Reply:
#95